gef.py

```diff
@@ -7792,18 +7817,31 @@ def __get_current_block_start_address():
                         if op in extended_registers[exreg]:
                             parameter_set.add(exreg)
 
-        if is_x86_32():
-            nb_argument = len(parameter_set)
-        else:
-            nb_argument = 0
-            for p in parameter_set:
-                nb_argument = max(nb_argument, function_parameters.index(p)+1)
+        nb_argument = -1
+        if function_name.endswith('@plt'):
+            _function_name = function_name.split('@')[0]
+            try:
+                nb_argument = len(libc_args_definitions[_function_name])
+            except:
+                pass
+
+        if nb_argument < 0:
```

Comment:
`if nb_argument is None`

Reply:
this would break line 7833:
nb_argument = max(nb_argument, function_parameters.index(p)+1)

Reply:
that's done in a loop. We could do this

```python
nb_argument = max(function_parameters.index(p)+1 for p in parameter set)
```